### PR TITLE
Update views

### PR DESCRIPTION
**Pull request by alejocano22**
update views

### CODE REVIEW

File: resources/lang/en/appointment.php [hunk 1]
```diff
@@ -22,4 +22,7 @@
   'update' => 'Update',
   'noAppointments' => 'You do not have any appointments yet',
 
+  //update
+  'updateApointment' => 'Update apointment',
```
Comment (sgiraldog) [inline resources/lang/en/appointment.php]: Appointment*

**Review by sgiraldog — CHANGES_REQUESTED** [review]
Great job, please change apointment to appointment

**Review by sgiraldog — APPROVED** [review]
Perfect

File: resources/lang/en/appointment.php (hunk 1 above)
Comment (alejocano22) [reply resources/lang/en/appointment.php]: Ok ok

Comment (alejocano22) [reply resources/lang/en/appointment.php]: Nos damos durisimo

**Pull request merged**